Update is_match and is_exact_match description (#26235)
DataDog/documentation: content/en/monitors/notify/variables.md

|----------------------------|--------------------------------------------------------------------|
| `{{#is_alert}}`            | The monitor alerts                                                 |
| `{{^is_alert}}`            | The monitor does not alert                                         |

> | `{{#is_match}}`            | The context matches the provided substring. If a numeric value is used it will be converted to a string.|

Reviewer: ```suggestion
| `{{#is_match}}`            | The context matches the provided substring. If a numeric value is used, it is converted to a string.|
```

| `{{^is_match}}`            | The context does not match the provided substring                  |

> | `{{#is_exact_match}}`      | The context exactly matches the provided string.<br> If a number is used, the numeric value will be considered, regardless of its type. This means that as long as two numbers have the same value, they will be considered equal by the function|

Reviewer: ```suggestion
| `{{#is_exact_match}}`      | The context exactly matches the provided string.<br> If a number is used, the numeric value is considered, regardless of its type. This means that as long as two numbers have the same value, they are considered equal by the function. |
```
